fw/hpack.c

```diff
@@ -833,7 +833,8 @@ tfw_hpack_add_index(TfwHPackDTbl *__restrict tbl,
 				T_DBG3("%s: dropped index: %u\n", __func__,
 				       early);
 				if (cp->last)
-					tfw_pool_clean(tbl->h_pool, cp->hdr);
+					tfw_pool_clean_single(tbl->h_pool,
```

Comment:
Previously the idea was to delete all chunks from ptr to the last one. Now we delete only single chunk. Can you please explain this change in the commit message